src/coreclr/src/vm/interoplibinterface.h

```diff
@@ -33,6 +34,26 @@ class ComWrappersNative
     static void DestroyManagedObjectComWrapper(_In_ void* wrapper);
     static void DestroyExternalComObjectContext(_In_ void* context);
     static void MarkExternalComObjectContextCollected(_In_ void* context);
+
+public: // COM activation
+    static void MarkWrapperAsComActivated(_In_ IUnknown* wrapper);
+};
+
+class GlobalComWrappers
+{
+public:
+    static void QCALLTYPE SetGlobalInstanceRegistered();
```

Comment:
Can you document that this should match the state of the private static member of the `ComWrapper` class?